src/Services/Flight/src/Flight/Aircrafts/Models/ValueObjects/ManufacturingYear.cs

```diff
@@ -0,0 +1,19 @@
+namespace Flight.Aircrafts.Models.ValueObjects;
+
+using System;
+
+public record ManufacturingYear : GenericValueObject<int>
```

Comment:
Please remove generic approach for this value object too.

Reply:
I'll work on implementing the recommended changes.

Reply:
done